synctl

```diff
@@ -71,7 +71,19 @@ def abort(message, colour=RED, stream=sys.stderr):
     sys.exit(1)
 
 
-def start(configfile, daemonize=True):
+def start(configfile: str, daemonize=True) -> bool:
```

Comment:
I suppose this should be 
```suggestion
def start(configfile: str, daemonize:bool=True) -> bool:
```

(or whatever black does with that)

Reply:
done